package.json

```diff
@@ -5,25 +5,19 @@
   "main": "index.js",
   "scripts": {
     "clean": "npm run clean:dist && npm run clean:coverage",
-      "clean:dist": "rimraf ./dist",
-      "clean:coverage": "rimraf ./coverage",
-
+    "clean:dist": "rimraf ./dist",
+    "clean:coverage": "rimraf ./coverage",
     "scripts": "npm run scripts:dev && npm run scripts:prod",
-      "scripts:dev": "webpack --config webpack.config.dev.js",
-      "scripts:prod": "webpack --config webpack.config.prod.js",
-
+    "scripts:dev": "webpack --config webpack.config.dev.js",
+    "scripts:prod": "webpack --config webpack.config.prod.js",
     "styles": "npm run styles:dev && npm run styles:prod",
-      "styles:dev": "node-sass --include-path ./node_modules/bourbon/app/assets/stylesheets/ src/sunchoke.full.scss dist/sunchoke.css",
-      "styles:prod": "node-sass --include-path ./node_modules/bourbon/app/assets/stylesheets/ --output-style compressed src/sunchoke.full.scss dist/sunchoke.min.css",
-
+    "styles:dev": "node-sass src/sunchoke.full.scss dist/sunchoke.css && ./node_modules/postcss-cli/bin/postcss -u autoprefixer -b 'last 2 versions' -r dist/*.css",
```

Comment:
I can not run this.
Error:
Rendering Complete, saving .css file...
Wrote CSS to C:\Users\cnguyen\DataPrep\sunchoke\dist\sunchoke.css
'.' n'est pas reconnu en tant que commande interne
ou externe, un programme exécutable ou un fichier de commandes.